### PR TITLE
Integrate client with native bucketing

### PR DESCRIPTION
**Pull request by rob-odwyer**
This adds the minimum integration with the DVCClient so that tests can run. Some tests are still failing, and the event queue tests have been configured to skip against the native implementation for now.

### CODE REVIEW

**Comment by rob-odwyer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #152** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/go-server-sdk/152" target="_blank">Graphite</a> 
    * **PR #154** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/go-server-sdk/154" target="_blank">Graphite</a> 
      * **PR #155** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/go-server-sdk/155" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/go-server-sdk/155?utm_source=stack-comment).

File: native_bucketing/config_manager.go [hunk 1]
```diff
@@ -3,11 +3,20 @@ package native_bucketing
...
+	"github.com/go-playground/validator/v10"
 )
 
 var internalConfigs = make(map[string]*configBody)
 var configMutex = &sync.RWMutex{}
 
+// use a single instance of Validate, it caches struct info
+var validate *validator.Validate
+
+func init() {
```
Comment (JamieSinn) [inline native_bucketing/config_manager.go:16]: does this get merged into the general "init" functionality for an app? or does it need explicit calling

Comment (rob-odwyer) [reply native_bucketing/config_manager.go:16]: It's an automatic Go thing - the runtime runs any `init` function in a package automatically on startup, so you can use it to initialize package-level vars like this.

Comment (JamieSinn) [reply native_bucketing/config_manager.go:16]: Yeah, I was always curious whether that worked for libs or just the top level executed package

**Review by chris-hoefgen — APPROVED** [review]

**Pull request merged**